packages/rocketchat-api/server/v1/misc.js

```diff
@@ -168,3 +168,22 @@ RocketChat.API.v1.addRoute('shield.svg', { authRequired: false }, {
 		};
 	}
 });
+
+RocketChat.API.v1.addRoute('spotlight', { authRequired: true }, {
+	get() {
+		check(this.queryParams, {
+			query: String
+		});
+
+		const { query } = this.queryParams;
+
+		const result = Meteor.runAsUser(this.userId, () =>
+			Meteor.call('spotlight', query, null, {
```

Comment:
Just a small change, but the `{ rooms: true, users: true }` isn't required since that's the default... so it could just be `Meteor.call('spotlight, query)` :thinking: